packages/@o3r/rules-engine/builders/rules-engine-extractor/helpers/rules-engine.extractor.ts

```diff
@@ -251,7 +263,7 @@ export class RulesEngineExtractor {
                     fact = {
                       ...fact,
                       type: 'string',
-                      // eslint-disable-next-line @typescript-eslint/no-base-to-string
+                      // eslint-disable-next-line @typescript-eslint/no-base-to-string -- convert non-null values to string
                       enum: schema.enum?.filter((value): value is JSONSchema7Type => !!value).map((v) => v!.toString())
```

Comment:
Does `JSONSchema7Type` has a valid toString method?

Reply:
we may need to rewrite this line in another PR, but this is not the goal of this PR